packages/embark/src/lib/core/events.js

```diff
@@ -101,8 +101,11 @@ EmbarkEmitter.prototype.request2 = function() {
   warnIfLegacy(requestName);
   if (this._events && !this._events['request:' + requestName]) {
     log("NO REQUEST LISTENER", requestName);
-    console.log("made request without listener: " + requestName);
-    console.trace();
+
+    if (process && process.env && process.env.DEBUGEVENTS) {
```

Comment:
`process && process.env && process.env.DEBUGEVENTS` is repeated a number of times, maybe it could be made more DRY with a function in `embark-utils`?

```js
function debugEventsEnabled() {
  return !!global?.process?.env?.DEBUGEVENTS;
}
```

There's support for [optional chaining](https://github.com/TC39/proposal-optional-chaining) syntax in our [babel config](https://github.com/embark-framework/embark/blob/master/babel.config.js#L29).

Reply:
If/when #1987 is merged, can use `globalThis` instead of `global`:

```js
function debugEventsEnabled() {
  return !!globalThis?.process?.env?.DEBUGEVENTS;
}
```

Reply:
Also, this condition is useless. Just use our logger, it's pretty much what iT's created for.
`this.logger.debug` for the "made request without listener"
`this,logger.trace` for the trace

Reply:
Just thinking.... I guess we *might* occasionally want to see event debugging output apart from other embark debug/trace output. I'm not sure, actually. Thoughts?